### PR TITLE
added new editions of tlg0090

### PR DESCRIPTION
**Pull request by ChiaraPalladino**
added several new editions. @sonofmun  the text is the same throughout, but have a look at the headers to see if everything is okay.

### CODE REVIEW

**Comment by ChiaraPalladino** [comment]
@sonofmun Hook gives an error on metadata availability on the internal cts file. I was not able to trace what that is.

File: data/tlg0090/tlg001/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            Geographica antiqua. Agathemeri Hypotyposis geographiae omnia Graeco-Latina. 
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Gronovius, Jacobus (1645-1716), editor and translator. 
+        </ti:description>
+    </ti:edition>
+    
+    
+    
+    <ti:translation urn="urn:cts:greekLit:tlg0090.tlg001.opp-eng1" workUrn="urn:cts:greekLit:tlg0090.tlg001">
```
Comment (sonofmun) [inline data/tlg0090/tlg001/__cts__.xml]: I think the problem is that you are required to have an xml:lang attribute on a `<ti:translation>` element. And several of your translations don't have that. Change that and see if it passes. But you are certainly right that that is not a very helpful error message.

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-eng1.xml [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+                    <persName xml:id="DDD">Digital Divide Data</persName>
+                    <resp>Corrected and encoded the text</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Gregory Crane</persName>
+                    <resp>Editor-in-Chief, Perseus Digital Library</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Matt Munson</persName>
+                    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
```
[the remark this answers is not in the document]
Comment (AlisonBabeu) [reply data/tlg0090/tlg001/tlg0090.tlg001.opp-eng1.xml]: Man, that is depressing man, we have to change your dates! :(

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-eng1.xml [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+                </respStmt>
+                <respStmt>
+                    <persName>Annette Gessner</persName>
+                    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
```
[the remark this answers is not in the document]
Comment (ChiaraPalladino) [reply data/tlg0090/tlg001/tlg0090.tlg001.opp-eng1.xml]: aaahhh too bad when the project personnel gets updates!! I'll go check those.

**Comment by ChiaraPalladino** [comment]
@sonofmun changes done. let me know if anything else is needed.

**Pull request merged**